lib/index.js

```diff
@@ -134,35 +245,67 @@ function getFastestFunction(port) {
   }
 }
 
+function _shouldUseSpawnSync(input) {
+  // use spawnSync (original sync-rpc method for sending request) for:
+  //  1. setup module (.t == 1)
+  //  2. normal request (.t == 0) and the request is an api call. (some server api calls fail with writing to keep-alive channel, like Pkg.writeContents.)
+  return input.t === 1 || input.t === 0 && input.a.u.indexOf('/typesys/8/') === -1;
```

Comment:
limit the keep-alive channel to only serve type loading requests (with `/typesys/8/` path)